client/components/pagination/pagination-page.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+class PaginationPage extends Component {
+	static propTypes = {
+		pageNumber: React.PropTypes.node.isRequired,
+		currentPage: React.PropTypes.number.isRequired,
+		totalPages: React.PropTypes.number.isRequired,
+		pageClick: React.PropTypes.func.isRequired,
+	}
+
+	clickHandler = ( event ) => {
```

Comment:
Again, parentheses are optional here.

Also, in this function body, we could destructure `this.props` for clarity:

```es6
const {
  currentPage,
  pageClick,
  pageNumber,
  totalPages,
} = this.props;

switch ( pageNumber ) { ... }
```